.github/workflows/chart-lint-test.yml

```
@@ -0,0 +1,52 @@
+name: Chart Lint and Test
+
+on:
+  push:
+    paths:
+      - 'deploy/helm/**'
+    branches:
+      - develop
```

zfs-localpv uses `master` branch
```suggestion
      - master
```